src/oge/validation.py

```diff
@@ -320,6 +362,9 @@ def check_non_missing_cems_co2_values_unchanged(cems_original, cems):
 
 def check_removed_data_is_empty(cems):
     """Checks that the rows removed by `data_cleaning.remove_cems_with_zero_monthly_data()` don't actually contain non-zero data"""
+    print("############# Begin validation.py:353 #############")
```

Comment:
These print statements should be removed

Reply:
I was still evaluating the outcome